apps/export/analyses/excel_exporter.py

```diff
@@ -0,0 +1,51 @@
+from django.core.files.base import ContentFile
+from django.conf import settings
+
+from export.formats.xlsx import WorkBook
+
+import logging
+
+logger = logging.getLogger('__name__')
+
+
+class ExcelExporter:
+    def __init__(self, analytical_statement_entries):
+        self.wb = WorkBook()
+        self.split = None
+        self.analysis_sheet = self.wb.get_active_sheet().set_title('Analysis')
+
+        self.titles = [
+            'Analysis Pillar ID',
+            'Analysis Pillar',
+            'Assignee',
+            'Statement ID',
+            'Statement',
+            'Entry ID',
+            'Entry',
+            'Entry Link',
+            'Source Link'
+        ]
+
+    def add_analytical_statement_entries(self, analytical_statement_entries):
+        self.analysis_sheet.append([self.titles])
+        for analytical_statement_entry in analytical_statement_entries:
+            url = f'{settings.HTTP_PROTOCOL}://{settings.DEEPER_FRONTEND_HOST}/permalink/projects/'
```

Comment:
Let's create a function for this. Maybe define a file like deep/premalinks.py and define permalink patterns and functions there.